webapp/api/controllers/InsuranceClaimController.js

```diff
@@ -252,24 +254,46 @@ module.exports = {
 
     finalizeInsuranceClaim: (req, res) => {
         getFinalizedClaim(req, res);
-    }
+    },
 
-    // Just wanted to add dump data to db 
-    // await Vet.create({
-    //     emailAddress: 'vet@ma.om',
-    //     password: 'patrik',
-    //     first_name: 'Patrik',
-    //     last_name: 'Skulavy',
-    //     address: 'J. Holleho 13, Tvrdosovce u Dominika doma',
-    //     phone_number: '0944522011',
-    // });
-    // await Vet.create({
-    //     emailAddress: 'v@ma.com',
-    //     password: 'vet',
-    //     first_name: 'Bethoven',
-    //     last_name: 'Ruzovy',
-    //     address: 'J. Holleho 13, Tvrdosovce u Dominika doma',
-    //     phone_number: '0944522011',
-    //     bank_account: 'SK110092200025125663642'
-    // });
-}
+    getReport: (req, res) => {
+      const fileName = req.param('fileName');
+
+      let file = require('path').resolve('/home/dominik/Repos/pis-insurance/webapp/.tmp/uploads/' + fileName + '.pdf');
```

Comment:
Isn't there a better way abstractly target files, so that code would work on all machines? e.g. setting environment variable?

Reply:
I already did something like that but it is in a different branch(you can specify an upload na download directory).

Reply:
https://github.com/tomasMizera/pis-insurance/blob/refactoring/webapp/api/controllers/InsuranceClaimController.js